content/en/tracing/trace_collection/library_config/nodejs.md

```diff
@@ -75,6 +75,10 @@ Enable automatic injection of trace IDs in logs for supported logging libraries.
 `DD_TRACE_RATE_LIMIT`
 : Percentage of spans to sample as a float between `0` and `1`. Defaults to `100` when `DD_TRACE_SAMPLE_RATE` is set. Otherwise, delegates rate limiting to the Datadog Agent.
 
+`DD_TRACE_SAMPLING_RULES` 
+: **Default**: `[{"sample_rate": 1.0}]` <br>
```

Comment:
Is this `<br>` intentional? I believe it is not needed. None of the other lines have it. I previewed the page locally, and it doesn't appear to cause any problems, but it might be a good idea to remove it just in case.

Reply:
No idea. I just copy/pasted this from the cpp docs.

Reply:
Seems to be correct. Anywhere with a `Default` section needs the `<br>` to bump the content to the next line.